core/include/traccc/edm/cell.hpp

```diff
@@ -74,6 +74,11 @@ struct cell_module {
 
 };  // struct cell_module
 
+struct cell_counter {
+    geometry_id module = 0;
+    unsigned int nCells = 0;
+};  // struct cell_counter
```

Comment:
Should this really be here? :thinking: For now I'd rather just put it into the CSV reading code itself. Until we need to use it in more places it's not worth it putting it into such a public place.